README.md

```diff
@@ -23,20 +23,20 @@ You have two options to install the verifier.
 
 #### Option 1: Install via go
 ```
-$ go install github.com/slsa-framework/slsa-verifier@v1.2.0
+$ go install github.com/slsa-framework/slsa-verifier@v1.2.1
```

Comment:
1.2.1 doesn't exist yet, maybe it does for the builder?

Reply:
my bad. Fixed.